core/solver/fcg.hpp

```diff
@@ -93,106 +82,59 @@ class Fcg : public BasicLinOp<Fcg<ValueType>>, public PreconditionedMethod {
         return system_matrix_;
     }
 
-    /**
-     * Gets the maximum number of iterations of the FCG solver.
-     *
-     * @return  The maximum number of iterations.
-     */
-    int get_max_iters() const noexcept { return max_iters_; }
-
-    /**
-     * Gets the relative residual goal of the solver.
-     *
-     * @return  The relative residual goal.
-     */
-    remove_complex<value_type> get_rel_residual_goal() const noexcept
+    std::shared_ptr<const LinOp> get_preconditioner() const
```

Comment:
Missing documentation (although it's trivial).

Reply:
Done. For some reason it doesn't show this one as outdated.